src/common/constants.js

```diff
@@ -2803,1167 +2803,4240 @@ const days = {
 };
 
 const vaMedicalFacilities = {
-  AK: [
-    { value: '463GA', label: 'FAIRBANKS VA CLINIC' },
-    { value: '463GB', label: 'KENAI VA CLINIC' },
-    { value: '463GC', label: 'MAT-SU VA CLINIC' },
-    { value: '463GD', label: 'HOMER VA CLINIC' },
-    { value: '463GE', label: 'JUNEAU VA CLINIC' },
+  "VT": [
```

Comment:
Sad that the keys of this object are no longer are in alphabetical order :(

I'm also curious how you got data out of the Excel doc any into this format.

Reply:
i wrote this script based on josh's instructions to extract the facilities from a few csv files

https://gist.github.com/lihanli/bc08346d1674943d0997dc48e0c2552a